packages/@aws-cdk/aws-lambda-nodejs/package.json

```diff
@@ -46,6 +46,18 @@
     "build+test": "npm run build && npm test",
     "compat": "cdk-compat"
   },
+  "cdk-build": {
+    "pre": [
+      "nohup /usr/local/bin/dockerd --host=unix:///var/run/docker.sock --host=tcp://0.0.0.0:2375 --storage-driver=overlay&",
```

Comment:
Wait what? Are we starting docker before the build of this package? Why is this necessary?

Reply:
This was an attempt to start docker in a non privileged mode

Reply:
Can we revert this? 

We are adding docker support in our build environment

Reply:
Reverted